### ISSUE TITLE
Accessibility:

### ISSUE THREAD

**Issue by Shubash-singh** [AxLupiN/AgroMarket#6]
App lacked accessibility features for users with disabilities.

**Comment by rosen911** [comment]
Data Integration and scability
